### ISSUE TITLE
As a guardian I would like to be able to add a new activity on a citizens week plan so that it fits their schedule

### ISSUE THREAD

**Issue by Tekrus** [aau-giraf/weekplanner-OLDUI#52]
Tasks: 

- [ ] Make a Pictogram search widget (or if one exists, use it)
- [ ] Make a BLoC that should have the responsibility of adding a activity(Pictogram) to a weekday, remember that a weekday belongs to a citizen. This BLoC could have both the obligation to send the http request to the web-api and stream the result to a listener.
- [ ] Make the weekplan-screen allow for the click on the plus button. (This might requirer a BLoC in order to keep track of state)

This story is blocked by #46

**Comment by sipman** [comment]
This feature surrounds the ability to click the "+"-button on the weekplan, and then see a screen from where I can search for a pictogram. When I've found my pictogram I should be able to click it and see the pictogram appear at the bottom of the "day"-list I clicked the "+"-button on — referer to the PO-groups prototypes for any UI questions.

Main activities with in this feature:
- Make a Pictogram search widget (or if one exists, use it)
- Make a BLoC that should have the responsibility of adding a activity(Pictogram) to a weekday, remember that a weekday belongs to a citizen. This BLoC could have both the obligation to send the http request to the web-api and stream the result to a listener.
- Make the weekplan-screen allow for the click on the plus button. (This might requirer a BLoC in order to keep track of state)
